packages/connect-ui/src/lib/events.ts

```diff
@@ -0,0 +1,11 @@
+import type { AuthResult, ConnectUIEventClose, ConnectUIEventConnect } from '@nangohq/frontend';
+
+export function triggerClose() {
+    const event: ConnectUIEventClose = { type: 'close' };
+    parent.postMessage(event, '*');
```

Comment:
where does parent come from?

Reply:
parent is the opener (the whole connect ui is in an iframe)